app/src/main/java/com/kolo/gorskih/tica/imagine/ui/main/MainActivity.kt

```diff
@@ -33,9 +51,23 @@ class MainActivity : AppCompatActivity() {
             val filePath = data?.getStringExtra(KEY_PATH) ?: ""
 
             if (filePath.isNotBlank()) {
-                Glide.with(this)
+                //TODO: define where this goes
+                /*Glide.with(this)
                     .load(filePath)
-                    .into(capturedImage)
+                    .into(capturedImage)*/
+            }
+        }
+    }
+}
+
+class MainActivityController : TypedEpoxyController<MainUI>(){
+    override fun buildModels(data: MainUI) {
+        for(i in 1..20)
```

Comment:
waht

Reply:
Stavio epoxy hahaha, ovaj for je samo za mock